docs/source/customizations/overview.mdx

```diff
@@ -1,21 +1,23 @@
 ---
-title: Apollo Router Customizations
+title: Router Customizations
 subtitle: Extend your router with custom functionality
-description: Extend the Apollo Router with custom functionality. Understand the request lifecycle and how customizations intervene at specific points.
+description: Extend the GraphOS Router or Apollo Router Core with custom functionality. Understand the request lifecycle and how customizations intervene at specific points.
 ---
 
-You can create **customizations** for the Apollo Router to add functionality that isn't available via built-in [configuration options](../configuration/overview/). For example, you can make an external call to fetch authentication data for each incoming request.
+You can create **customizations** for the GraphOS Router or Apollo Router Core to add functionality that isn't available via built-in [configuration options](../configuration/overview/). For example, you can make an external call to fetch authentication data for each incoming request.
 
 ## Customization types
 
-The Apollo Router supports the following customization types:
+The GraphOS Router supports the following customization types:
```

Comment:
Available in both Router Core and GraphOS Router

Reply:
This already states that GraphOS Router supports both Rhai and coproc, while Apollo Router Core supports only Rhai.